### PR TITLE
Integrate Lerna for packages release orchestration - v1

### PR DESCRIPTION
**Pull request by dprokop**
#17981

### CODE REVIEW

**Comment by dprokop** [comment]
@ryantxu syncing all packages versions to v6.3.0-alpha.30 - to follow our toolkit's crazy versioning ;)

File: packages/grafana-runtime/rollup.config.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ const buildCjsPackage = ({ env }) => {
         globals: {},
       },
     ],
-    external: ['lodash'], // Use Lodash from grafana
+    external: ['lodash', '@grafana/ui'], // Use Lodash from grafana
```
Comment (ryantxu) [inline packages/grafana-runtime/rollup.config.ts]: and data?

**Comment by ryantxu** [comment]
Another issue we need to deal with is how packages depend on one another.
```
runtime -> depends on -> ui -> depends on -> data
```
BUT we don't want the deps in main package.json since that is suppored via workspaces

In our publish scripts, can we add the dependency with the right version (since they will all share version at release)

File: packages/grafana-runtime/package.json [hunk 2]
```diff
@@ -36,5 +37,6 @@
   },
   "resolutions": {
     "@types/lodash": "4.14.119"
-  }
+  },
+  "gitHead": "eef02dc9b15ddd8923ec3571955a47e0f40d6562"
```
Comment (ryantxu) [inline packages/grafana-runtime/package.json]: this should not be in the versioned file, right?  just the published one

Comment (dprokop) [reply packages/grafana-runtime/package.json]: Yes, good catch

File: packages/grafana-toolkit/src/cli/tasks/grafanaui.release.ts [hunk 3]
```diff
@@ -1,190 +1,191 @@
...
-      chalk.green(
-        `\nVersion ${nextVersion} of ${pkg.name} succesfully prepared for release. See packages/grafana-ui/dist`
-      )
-    );
-    console.log(chalk.green(`\nTo publish to npm registry run`), chalk.bold.blue(`npm run gui:publish`));
-  }
-};
-
-export const releaseTask = new Task<ReleaseTaskOptions>('@grafana/ui release', releaseTaskRunner);
+// import execa = require('execa');
```
Comment (dprokop) [inline packages/grafana-toolkit/src/cli/tasks/grafanaui.release.ts]: TODO: Remove this

**Review by ryantxu — APPROVED** [review]
LGTM

**Pull request merged**